doc/tutorials/mount.md

```diff
@@ -1,56 +1,146 @@
+    $ cat /etc/hosts
+    ...
+    mylocalhost 127.0.0.33       localhost
     ...
 
-Thus we wrote using the format used by all applications, applications
-will now pick up these changes:
+Applications will now pick up these changes:
+
+    $ ping mylocalhost
+
+###### Why you need superuser privileges to mount files ######
+Elektra manages its mountpoints in configuration below **system/elektra/mountpoints**.
+The file that holds this configuration is per default only writable by administrators:
+
+    $ kdb file system/elektra/mountpoints
+    /etc/kdb/elektra.ecf
+
+Because of that only root users can mount files.
+
+> At the moment a workaround for this is to add your user
+> to a group that can modify Elektras configuration file.
+>
+>     $ sudo groupadd elektra
+>     $ KDB_SYSTEM=$(dirname $(kdb file system))
+>     $ sudo chown -R :elektra $(dirname $(kdb file system))
+>     $ sudo chmod -R g+w $(dirname $(kdb file system))
+>
+> Bear in mind that users in this group con modify all configuration of Elektra
+> that is stored in files the user is permitted to write.
+
+## Resolver ##
+
+The configuration file path you supply to `kdb mount` is actually not an
+absolute or relative path in your filesystem, but gets resolved to one by Elektra.
+The plugin that is responsible for this is the _Resolver_.
+
+When you mount a configuration file the resolver first looks at the namespace of
+your mountpoint. Based on that namespace and if the supplied path was relative or
+absolute the resolver than resolves the supplied path to a path in the file system.
+
+If you supplied an absolute path (e.g. `/example.ini`) it gets resolved to this:
+
+| Namespace        | resolved Path       |
+| ---------------- |-------------------- |
+| spec             | /example.ini        |
+| dir              | ${PWD}/example.ini  |
+| user             | ${HOME}/example.ini |
+| system           | /example.ini        |
+
+If you supplied a relative path (e.g. `example.ini`) it gets resolved to this:
+
+| Namespace        | resolved Path               |
+| ---------------- |---------------------------- |
+| spec             | /usr/share/elektra/specification/example.ini |
+| dir              | ${PWD}/.dir/example.ini     |
+| user             | ${HOME}/.config/example.ini |
+| system           | /etc/kdb/example.ini        |
+(If this differs on your system, the resolver has a different configuration. You
+can then check the default paths under `system/elektra/modules/resolver/constants`)
+
+## Plugins ##
+
+Configuration files can have many different formats (ini, json, yaml, xml, csv, ... to name but a few).
+
+One of the goals of Elektra is to provide users with a unified interface to all those formats.
+Elektra accomplishes this task with _storage plugins_.
+When you mount a file you can tell Elektra which plugins it should use for reading and
+writing to configuration files.
+
+Lets mount a projects git configuration into the dir namespace:
```

Comment:
> README in the tutorial-folder ...

This can be changed, too. We also need to links at the end of tutorials to the next.